json-endpoints/jetpack/class.jetpack-json-api-themes-install-endpoint.php

```diff
@@ -14,6 +14,12 @@ protected function install() {
 
 		foreach ( $this->themes as $theme ) {
 
+			// hook to allow alternative install method
+			if ( wp_endswith( $theme, '-wpcom' )
+				 && apply_filters( 'jetpack_wpcom_theme_install', false, $theme ) ) {
+				continue;
```

Comment:
If we `continue` at this place, the logger on line `47` (`$this->log[ $theme ][] = $upgrader->skin->get_upgrade_messages();`) won't run and so won't log that the theme was successfully installed.